teamcity-rest-client-api/src/main/kotlin/org/jetbrains/teamcity/rest/api.kt

```diff
@@ -199,6 +205,8 @@ interface Change {
      * Web UI URL for user, especially useful for error and log messages
      */
     fun getWebUrl(specificBuildConfigurationId: BuildConfigurationId? = null, includePersonalBuilds: Boolean? = null): String
+
+    fun firstBuilds(): List<Build>
```

Comment:
Meaning of this method is unclear. How many first builds will it return? I think it's worth to add javadoc to it.

Reply:
It's indeed unclear as it uses [the undocumented experimental feature](https://youtrack.jetbrains.com/issue/TW-24633). I'll add a JavaDoc which mentions the issue.